src/main/resources/roleconfig/role_common.json

```diff
@@ -2553,5 +2697,71 @@
         }
       }
     ]
+  },
+  {
+    "name": "regional-centre-team-leader",
+    "label": "Regional Centre Team Leader",
+    "description": "Regional Centre Team Leader role for admin users",
+    "category": "ADMIN",
+    "substantive": true,
+    "type": "ORGANISATION",
+    "patterns": [
+      {
+        "roleType": {
+          "mandatory": true,
+          "values": ["ORGANISATION"]
+        },
+        "grantType": {
+          "mandatory": true,
+          "values": ["STANDARD"]
+        },
+        "classification": {
+          "mandatory": true,
+          "values": ["PUBLIC"]
+        },
+        "attributes": {
+          "jurisdiction": {
+            "mandatory": true,
+            "values": ["EMPLOYMENT","SSCS"]
+          },
+          "primaryLocation": {
+            "mandatory": true
+          }
+        }
+      }
+    ]
+  },
+  {
+    "name": "regional-centre-admin",
+    "label": "Regional Centre Admin",
+    "description": "Regional Centre Admin role for admin users",
+    "category": "ADMIN",
+    "substantive": true,
+    "type": "ORGANISATION",
+    "patterns": [
+      {
+        "roleType": {
+          "mandatory": true,
+          "values": ["ORGANISATION"]
+        },
+        "grantType": {
+          "mandatory": true,
+          "values": ["STANDARD"]
+        },
+        "classification": {
+          "mandatory": true,
+          "values": ["PUBLIC"]
+        },
+        "attributes": {
+          "jurisdiction": {
+            "mandatory": true,
+            "values": ["EMPLOYMENT","SSCS"]
```

Comment:
`"regional-centre-admin"` needs a unit test in AllServicesOrgRoleTest.java/shouldApproveRequestedRoleForOrg

Reply:
added in 777f518